src/MobileAPIServiceProvider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace ColoredCow\LaravelMobileAPI;
+
+use Illuminate\Support\ServiceProvider;
+use Laravel\Passport\Passport;
+
+class MobileAPIServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        Passport::routes();
```

Comment:
Just to make this library more generic let's remove this service provider from here and this in the portal engine package service provider.

Reply:
@pankaj-ag done